### PR TITLE
fix error info for transpose sequence_conv_pool max_sequence_len sequ…

### PR DESCRIPTION
**Pull request by 1024er**
fix error info for transpose sequence_conv_pool max_sequence_len sequence_erase

### CODE REVIEW

File: paddle/fluid/operators/sequence_ops/sequence_erase_op.h [hunk 1]
```diff
@@ -30,9 +30,13 @@ class SequenceEraseKernel : public framework::OpKernel<T> {
...
         lod.empty(), false,
-        "Input(X) Tensor of SequenceEraseOp does not contain LoD information.");
-    PADDLE_ENFORCE_EQ(lod[lod.size() - 1].back(), (size_t)in->numel(),
-                      "The actual size mismatches with the LoD information.");
+        platform::errors::InvalidArgument("Input(X) Tensor of SequenceEraseOp "
+                                          "does not contain LoD information."));
+    PADDLE_ENFORCE_EQ(
+        lod[lod.size() - 1].back(), (size_t)in->numel(),
+        platform::errors::InvalidArgument(
+            "The actual size %d mismatches with the LoD information size %d.",
```
Comment (chenwhql) [inline paddle/fluid/operators/sequence_ops/sequence_erase_op.h]: `actual size` -> `actual input size`?

File: paddle/fluid/operators/transpose_op.h [hunk 2]
```diff
@@ -53,7 +53,10 @@ inline void TransCompute(const int dim, const DeviceContext& dev_ctx,
       trans6(dev_ctx, in, out, axis);
       break;
     default:
-      PADDLE_THROW("Tensors with rank at most 6 are supported");
+      PADDLE_THROW(
```
Comment (chenwhql) [inline paddle/fluid/operators/transpose_op.h]: need with error type, like `PADDLE_THROW(platform::errors::Unimplemented(...))`

File: paddle/fluid/operators/sequence_ops/sequence_erase_op.h (hunk 1 above)
Comment (1024er) [reply paddle/fluid/operators/sequence_ops/sequence_erase_op.h]: fixed

File: paddle/fluid/operators/transpose_op.h (hunk 2 above)
Comment (1024er) [reply paddle/fluid/operators/transpose_op.h]: Added platform::errors::InvalidArgument(...)

**Review by kuke — APPROVED** [review]
LGTM

**Pull request merged**